corehq/apps/app_manager/templates/app_manager/download_index.html

```diff
@@ -53,6 +53,13 @@ <h2>App Profiles</h2>
   {% endif %}
 {% endblock %}
 
+{% block mm_block %}
```

Comment:
This page is getting crowded. It'd be nice to split it into tabs - I think each of the current `&lt;h2&gt;` sections would be a tab. What do you think?

Reply:
content is very small per section, so i think its fine to leave it as-is.
And actually i think we can now remove "App Profiles" and other things that were added for app profile restriction since now we have added a new page to manage that

Reply:
Removing things sounds good :)

Reply:
Cool, will check with GS and remove it